processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -511,6 +522,19 @@ public boolean isStatic() {
         return executable.getModifiers().contains( Modifier.STATIC );
     }
 
+    @Override
+    public boolean isDefault() {
+        boolean isDefault = false;
+        if ( mapperToImplement != null ) {
```

Comment:
AFAIK the `SourceMethod` from which these are taken can also be referenced methods, right? They don't require implementation (e.g. can be hand written).